src/modules/add-and-remove.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable import/prefer-default-export */
+import { Task } from './to-to';
+import { input } from './html-element';
+// js for add and delete functionalities
```

Comment:
- [ ] Please, consider removing these comments from everywhere to make your code clear. Your code is understandable and it tells by itself what happening 👍

Reply:
ok sir.